### PR TITLE
PR template for blog posts and development work to improve our blog

### PR DESCRIPTION
**Pull request by koebel**
adding two templates for submitting PRs to our blog
- `BLOG_POST_PULL_REQUEST_TEMPLATE.md` is for submitting a blog post
- `DEVELOPMENT_PULL_REQUEST_TEMPLATE.md` is for submitting some development improvements in our blog

### CODE REVIEW

File: .github/DEVELOPMENT_PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+- [ ] Breaking change (fix or feature that would cause existing functionality to change)
+- [ ] Technical debt
+- [ ] Tests
+- [ ] Documentation
+- [ ] Maintenance (e.g. dependency updates or tooling)
+
+## Checklist:
+- [ ] Code changes
+- [ ] Unit tests added
+- [ ] Acceptance tests added
```
Comment (phil-davis) [inline .github/DEVELOPMENT_PULL_REQUEST_TEMPLATE.md:40]: Do we have both these levels of automated testing in the blog repo?

Comment (koebel) [reply .github/DEVELOPMENT_PULL_REQUEST_TEMPLATE.md:40]: it seems like so far we don't have any tests our blog... maybe we should add some considering that this is our expertise :)

Shall we keep these points in the template or remove them until we actually do add some tests?

Comment (phil-davis) [reply .github/DEVELOPMENT_PULL_REQUEST_TEMPLATE.md:40]: I guess that we can keep this, and it will remind us to think about automated tests.

**Review by phil-davis — APPROVED** [review]

File: .github/DEVELOPMENT_PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (koebel) [reply .github/DEVELOPMENT_PULL_REQUEST_TEMPLATE.md:40]: opened an issue for this, see https://github.com/JankariTech/blog/issues/131

**Review by individual-it — APPROVED** [review]

**Review by SagarGi — APPROVED** [review]

**Comment by SagarGi** [comment]
@koebel could you sign all your commits?

**Review by SagarGi — APPROVED** [review]

**Comment by koebel** [comment]
> @koebel could you sign all your commits and then only we can merge it.

Thanks for the hint. I've added a GPG key to my GitHub, but it doesn't seem to affect previous commits. @SagarGi do you know if that will be required for merging? 

@individual-it I think I dismissed your review by mistake while trying to fix the issue mentioned above...

**Pull request merged**